src/hotspot/os/bsd/os_bsd.hpp

```diff
@@ -70,7 +70,7 @@ class os::Bsd {
   // Real-time clock functions
   static void clock_init(void);
 
-  static void *dlopen_helper(const char *path, int mode);
+  static void *dlopen_helper(const char *path, int mode, char *ebuf, int ebuflen);
```

Comment:
Per-existing, but these helpers should not have been exposed via os::Bsd or os::Linux, they should be filescope static.